reference-cni-plugin-install/reference-cni-daemonset.yml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cni-binary-copy-script
+  namespace: kube-system
+  annotations:
+    kubernetes.io/description: |
+      This is a script used to copy CNI binaries
+data:
+  cnibincopy.sh: |-
+    #!/bin/bash
+    cp -f /usr/src/plugins/bin/* /host/opt/cni/bin
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-containernetworking-plugins-ds-amd64
+  namespace: kube-system
+  labels:
+    tier: node
+    app: containernetworking-plugins
+    name: containernetworking-plugins
+spec:
+  selector:
+    matchLabels:
+      name: containernetworking-plugins
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: containernetworking-plugins
+        name: containernetworking-plugins
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        kubernetes.io/arch: amd64
+      tolerations:
+      - operator: Exists
+        effect: NoSchedule
+      initContainers:
```

Comment:
I'm just wondering why do we need initContainer here? Can we use it as a regular container?